src/Core/Vipr.Core/OutputFile.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Vipr.Core
+{
+    public abstract class OutputFile
```

Comment:
@tonycrider please check this comment : https://github.com/Microsoft/Vipr/issues/54#issuecomment-86125992 

I like your proposal though, @piotrpMSFT what do you think?